editor/connections_dialog.cpp

```diff
@@ -619,6 +639,7 @@ void ConnectionsDock::_make_or_edit_connection() {
 	if (add_script_function) {
 		// Pick up args here before "it" is deleted by update_tree.
 		script_function_args = it->get_metadata(0).operator Dictionary()["args"];
+		script_function_args.resize(script_function_args.size() - cd.unbinds);
```

Comment:
What is this for? 🤔

Reply:
For argument list of auto-generated functions (`Receiver Method` in script). If the number of parameters sent is not equal to the number of parameters of the receiving method, an error will be reported.

Reply:
The unbinds spinbox will eventually decide how many parameters to unbind.
Its value is actually the unbind size of the newly constructed `callable`.
This error occurs if it is greater than 0 (the size of the arguments in `focus_entered` is `0`).
```
E 0:00:03:0853   emit_signalp: Error calling from signal 'focus_entered' to callable: VBoxContainer(test.gd)::_on_button_2_focus_entered : Method expected 0 arguments, but called with 0..
  <C++ Source>   core/object/object.cpp:1132 @ emit_signalp()
```

Reply:
> The unbinds spinbox could get disabled if max is 0, so users wouldn't wonder why it's non-functional.

The maximum value is `0`, which means there are `no` parameters to unbind, that is, the size of the defined signal parameter list is `0`, disable it is reasonable.

Reply:
Ah, so this basically fixes a bug. Nice.

Reply:
> Ah, so this basically fixes a bug. Nice.

Yes, it does.